### ISSUE TITLE
Tables on Benefits detail pages not displaying captions

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16725]
## Describe the defect
It looks like tables that were previously not using the `<va-table>` component were auto updated somewhat recently and when that happened the table caption from Drupal were not migrated correctly so they are now missing from the FE of VA.gov. [This issue was brought to our attention By Randi H.](https://dsva.slack.com/archives/CDHBKAL9W/p1704485014622559?thread_ts=1703102948.578279&cid=CDHBKAL9W)

For example, there is a [copay rates table on this page](https://www.va.gov/health-care/copay-rates/). Currently there is no table caption:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/c35a91fe-9786-4ab3-a733-544e3d6d69ad)

However, using the Way Back Machine, we can see that there used to be a caption:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/a46d1448-9dbc-4c2c-9e4e-5efc50348c82)

I looked at the component on the DS site and say that there is a prop for `table-title` that should be used for the caption and we do still have this info in Drupal. So I believe we just need to update the templates where we pull in tables to put the Drupal table caption into the `table-title` prop in the `<va-table>` component.

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/93b40906-1781-42b2-add3-6a9040664d0e)

### Additional Info
This might be happening for all instances of table, however for prioritizing Randi noted that getting this info resolved for the Benefits detail pages quickly would be the preference. Then after, we might need to look at of instances of table - I might also need to look at the Facilities tables as well

## AC / Expected behavior

- [ ] Table captions should display for the copay rates table mentioned above
- [ ] - [ ] Accessibility review
- [ ] Additional FE ticket is written to examine all other all tables on the FE to see if any others are missing the caption, and to then fix. (this ticket is limited in scope so as not to impact the Federal Header/Footer work being done)

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by FranECross** [comment]
@chriskim2311 If I were to pull this defect into the current sprint, would you be able to look at it/fix it without any impact to the Federal Header/Footer work? The scope of this ticket is limited to the table that  Randi Hecht indicated is negatively impactful to Veterans, and another ticket will be written to examine all other tables to determine if they need to be fixed as well, and if so, will be pulled into the following sprint. Thanks in advance! cc @laflannery @jilladams @rlhecht

**Comment by chriskim2311** [comment]
@FranECross Yes lets pull this ticket in and I can take a look at it thanks!

**Comment by FranECross** [comment]
> @FranECross Yes lets pull this ticket in and I can take a look at it thanks!

Thanks so much! Pulled in.  cc @jilladams

**Comment by chriskim2311** [comment]
@laflannery One more ticket to verify in prod thank you!

**Comment by laflannery** [comment]
@chriskim2311 I reviewed both pages that Randi H identified from the slack thread and these look good to me!

**Comment by FranECross** [comment]
@chriskim2311 Looks great! Thanks!

**Comment by chriskim2311** [comment]
Verified on prod. Closing!
